component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -1553,6 +1566,9 @@ private void logMetrics(ExoSocialActivity activity) {
 
   private void saveProfile(String username, ProfileEntity profileEntity) throws Exception {
     Identity userIdentity = getUserIdentity(username);
+    if(userIdentity == null) {
```

Comment:
This will return NULL if the user is anonymous or is not present -> IdentityManager is enabled to retrieve him. I am not sure if we need to recheck again in this function.